en/move.json

```
@@ -3317,7 +3317,7 @@
   },
   "stoneAxe": {
     "name": "Stone Axe",
-    "effect": "The user swings its stone axes at the target. Stone splinters left behind by this attack float around the target."
+    "effect": "The user swings large stone axes at the target. Stone splinters left behind by this attack float around the target."
```

```suggestion
    "effect": "The user swings a large stone axe at the target. Stone splinters left behind by this attack float around the target."
```

Having it specify only one axe makes it sound better if the intent is to make it more generic.